tidb-cloud/ticloud-serverless-spending-limit.md

```diff
@@ -0,0 +1,49 @@
+---
+title: ticloud serverless cluster spending-limit
+summary: The reference of `ticloud serverless spending-limit`.
+---
+
+# ticloud serverless spending-limit
+
+Set the maximum monthly [spending limit](/tidb-cloud/manage-serverless-spend-limit.md) for a TiDB Serverless cluster:
+
+```shell
+ticloud serverless spending-limit [flags]
+```
+
+## Examples
+
+Set the spending limit for a TiDB Serverless cluster in interactive mode:
+
+```shell
+ticloud serverless spending-limit
+```
+
+Set the spending limit for a TiDB Serverless cluster in non-interactive mode:
+
+```shell
+ticloud serverless spending-limit -c <cluster-id> --monthly <spending-limit-monthly>
+```
+
+## Flags
+
+In non-interactive mode, you need to manually enter the required flags. In interactive mode, you can just follow CLI prompts to fill them in.
+
+| Flag                    | Description                                 | Required | Note                                                 |
+|-------------------------|---------------------------------------------|----------|------------------------------------------------------|
+| -c, --cluster-id string | Specifies the ID of the cluster.                       | Yes      | Only works in non-interactive mode.                  |
+| --monthly int32         | Specifies the maximum monthly spending limit in USD cents. | Yes      | Only works in non-interactive mode.                  |
+| --force                 | Deletes a cluster without confirmation.      | No       | Works in both non-interactive and interactive modes. |
```

Comment:
"Deletes a cluster without confirmation." looks incorrect here for this command

Reply:
It shoud be deleted